xbout/tests/test_animate.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from xbout import open_boutdataset
+from xbout.boutdataarray import BoutDataArrayAccessor
+
+from animatplot.blocks import Imshow, Line
+
+# Path to test dmp files
+DATA_PATH = './xbout/tests/data/dump_files/along_x/BOUT.dmp.*.nc'
+
+
+@pytest.fixture
+def create_test_file(tmpdir_factory):
+
+    # Create temp dir for output of animate1D/2D
+    save_dir = tmpdir_factory.mktemp("test_data")
+    ds = open_boutdataset(DATA_PATH).squeeze(drop=True)  # Open test data
+
+    return save_dir, ds
+
+
+class TestAnimate:
+    """
+    Set of tests to check whether animate1D() and animate2D() are running properly
+    and PillowWriter is saving each animation correctly
+    """
+    def test_animate2D(self, create_test_file):
+
+        save_dir, ds = create_test_file
+        animation = ds['T'].bout.animate2D(y='z', save_as="%s/test.gif" % save_dir)
```

Comment:
It doesn't affect the function of the test, but a `'.gif'` suffix gets added to the file name by `animate2D`/`animate1D`, so this call will create a file named `test.gif.gif`. Please remove the `.gif` in the call here and below (in `test_animate1D`), just so that they show the intended usage of the `save_as` argument. Thanks!

Reply:
Thanks John. Apologies will fix it now!